source/user-manual/capabilities/system-inventory/compatibility-matrix.rst

```diff
@@ -25,3 +25,29 @@ The Syscollector module supports different options across various operating syst
+
+The following table shows the new feature that are compatible with various operating systems.
+
++------------------------+------------------------------------------------------------------------------------------------------------------------------------+
+|                        |                                        **Syscollector package enhancement**                                                        |
++  **Operating System**  +----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|                        |        NPM           |       PyPI      |       rpm       |       deb        |       pacman     |       pkg       |       .pkg      |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|    Windows             |                  √   |        √        |        –        |        –         |        –         |        –        |        –        |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|    Linux               |                  √   |        √        |        √        |        √         |        √         |        √        |        –        |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|    macOS               |                  √   |        √        |        –        |        –         |        –         |        –        |        –        |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|    FreeBSD             |                  √   |        √        |        –        |        –         |        –         |        –        |        –        |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|    OpenBSD             |                  √   |        √        |        –        |        –         |        –         |        –        |        –        |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
+|    Solaris             |                  √   |        √        |        –        |        –         |        –         |        –        |        √        |
++------------------------+----------------------+-----------------+-----------------+------------------+------------------+-----------------+-----------------+
```

Comment:
I don't understand why ".pkg" and FreeBSD also uses pkg package manager, Although I think they are different package manager that share the same name. 
![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/cbb8535b-7f6a-4e99-a2d6-926606acd0d3)

Reply:
Good catch ! Fix it !